[2025-07-28T19:31:21.073+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T19:31:21.097+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: financial_etl_pipeline.transform_data manual__2025_07_28_agg [queued]>
[2025-07-28T19:31:21.105+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: financial_etl_pipeline.transform_data manual__2025_07_28_agg [queued]>
[2025-07-28T19:31:21.106+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 3
[2025-07-28T19:31:21.263+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-28 18:46:50+00:00
[2025-07-28T19:31:21.271+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'transform_data', 'manual__2025_07_28_agg', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py', '--cfg-path', '/tmp/tmpeh6dek5h']
[2025-07-28T19:31:21.274+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2371) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T19:31:21.275+0000] {standard_task_runner.py:91} INFO - Job 182: Subtask transform_data
[2025-07-28T19:31:21.275+0000] {standard_task_runner.py:63} INFO - Started process 2375 to run task
[2025-07-28T19:31:21.342+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.transform_data manual__2025_07_28_agg [running]> on host 7815b21b8845
[2025-07-28T19:31:21.479+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='financial_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T18:46:50+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025_07_28_agg'
[2025-07-28T19:31:21.483+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T19:31:21.516+0000] {logging_mixin.py:188} INFO - Transformation complete and saved to curated CSV.
[2025-07-28T19:31:21.517+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-28T19:31:21.517+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T19:31:21.526+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=financial_etl_pipeline, task_id=transform_data, execution_date=20250728T184650, start_date=20250728T193121, end_date=20250728T193121
[2025-07-28T19:31:21.570+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-28T19:31:21.593+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-28T19:31:21.595+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
